### DISCUSSION TITLE
How are jedis release versions built?

### DISCUSSION THREAD

**Discussion by evantorrie** [xetorthio/jedis#2538] [Q&A]
I noticed that `v3.5.1` has release notes as "3.5.0 recompiled with JDK8".

I ask because we had an issue when we upgraded to `jedis-3.4.1` for our application and found that it suddenly started breaking on JDK8 runtimes.

I was looking through the github actions, circleci etc in this repo and I don't see where it actually publishes a release version - everything just seems to be SNAPSHOT builds in the automated CI.

**Comment by sazzad16** [comment]
Jedis releases between December 10, 2020 and January 20, 2021 are likely to have build issue(s). Please update to a more recent release.
